brion/plugin/compartmentReportBinary.cpp

```diff
@@ -204,7 +204,7 @@ size_t CompartmentReportBinary::getFrameSize() const
     return _subtarget ? _comps : _header.numCompartments;
 }
 
-floatsPtr CompartmentReportBinary::loadFrame( const float timestamp ) const
```

Comment:
I was getting the virtual override and I changed all the prototypes, both in .h and .cpp files. I guess I should have changed only the headers, right? I'll roll back cpp changes if so